doc/source/development/contributing.rst

```diff
@@ -949,6 +949,11 @@ the expected correct result::
 
         assert_frame_equal(pivoted, expected)
 
+.. note::
+
+   Add the GitHub Issue Number as a Comment to the New Tests.
```

Comment:
don't need to capitalizie new tests. Can you point to an example in the code, e.g.https://github.com/pandas-dev/pandas/blob/master/pandas/tests/series/test_apply.py#L166 (or you can just replicate this here maybe better)

Reply:
Ok
I will change that capitalized word
and for the example - Shall i add the following line 
'For Example : # GH 20714 bug fixed in: GH 24275" 
or anything else?

Reply:
yes that sounds good (make it clear its an example :->)

Reply:
Or should i just replicate like
'For Example : def test_apply_categorical_with_nan_values(self, series):
                                 # GH 20714 bug fixed in: GH 24275
                                 s = pd.Series(series, dtype="category")
                                 result = s.apply(lambda x: x.split("-")[0])'

Reply:
I'll be writing
For example - If you modify the test - "test_apply_categorical_with_nan_values" (GH 20714), please write-
def test_apply_categorical_with_nan_values(self, series):
# GH 20714 bug fixed in: GH 24275
s = pd.Series(series, dtype="category")

Reply:
Is that fine? 
Should i add the entire function or a few lines is enough for example?

Reply:
You can just add `GH XXXX` to the test that directly precedes this

Reply:
Can you give me an example. I'm not getting it.
What exactly should i write as an example.
Sorry for pestering

Reply:
See the test case in #28662 how it has # GH XXXX as a comment at the top of the test (you can find this in other PRs as well)

You can add that in the test case shown in this file

Reply:
I've added it in my latest commit.
Please let me know

Reply:
Please Check